package.json

```diff
@@ -77,6 +77,7 @@
     "full-icu": "^1.3.4",
     "generate-license-file": "^1.3.0",
     "gettext-extractor": "^3.5.2",
+    "get-func-name": "^v2.0.1",
```

Comment:
They get added as dependencies from other libraries. But the version defined on the other libraries are still lower than what we want so I specified the versions like this.

Reply:
I see. I don't think we need to add transitive deps to the package.json. Now that we've updated the package-lock, the repo no longer has the vulnerability. And, if a user installs the repo, they'll get the latest versions of all transitive deps anyways. If one of our deps pins to a specific version of the transitive dep with the issue, then we'd have to add the transitive dep to the package.json overrides section. But, it doesn't look like that's the case here

Reply:
I was trying to have something that specifically says what's the min version we want for the transitive deps. But I agree with your concern about any future dependency requirements change. Seems like `ansi-regex` is also a transitive dep. Maybe it's fine to keep them there?

Reply:
the reason the other deps (like ansi-regex) are there is because we're not able to update those via `npm update`. some of our other deps require a version of ansi-regex that still has the vulnerability. we have to force them to use a different version, by putting them in the overrides

I agree that it would be nice to have a place where we specify the min version, but I'm not too concerned because this won't be an issue for users setting up a new repo, and especially because these particular transitive deps are all dev dependencies and wouldn't be installed by a user anyways

Reply:
ok sounds good. Going to remove them!